README.md

```diff
@@ -1,28 +1,158 @@
 
-It scans over the network, and simply records the module and method for every event/extrinsic on each block - please see the standard entities in `schema.graphql`.
+You can start by familiarizing yourself with [Ternoa Explorer](https://explorer.ternoa.com/) as it was built using the Ternoa Dictionary.
 
-**If you want to create your SubQuery Dictionary to speed up indexing of your own Substrate chain, fork this project and let us know**
+Don't forget to have fun with it, Cheers 🍻
 
-# Geting Started
-### 1. Install dependencies
-```shell
-yarn
+**Table of Contents :** 
+
+- [Introduction](#🤔-introduction)
+  - [Dictionary](#dictionary)
+  - [Error Reporting](#error-reporting)
+- [Installation](#installation)
+  - [Clone the Repository](#1-clone-the-repository)
+  - [Change Directory](2-change-directory)
+  - [Select Testnet Version](#3-select-testnet-version)
+  - [Install Dependencies](#4-install-dependencies)
+  - [Generate Type from GraphQL](#5-generate-type-from-graphql)
+  - [Build your Implementation](#6-build-your-implementation)
+  - [Docker Pull](#7-docker-pull)
+  - [Run](#8-run)
+- [Using the Dictionary](#using-the-dictionary)
+  - [Examples](#examples)
+ - [Using Dictionary as an Explorer](#using-dictionary-as-an-explorer)
+
+
+## 🤔 Introduction
+
+**Ternoa is a Decentralised, Open source, NFT-centric Layer 1 blockchain that is multi-chain by design and aims to provide a technical stack to build scalable and secure NFTs with native support for advanced features.**
+
+#### For Builders By Builders
+
+NFTs native to our chain can be minted using High-level programming languages and doesn't require smart contract functionality.
+
+#### Native support for Advanced Features
+
+With native support for Secret NFTs, Delegating and Lending, Transaction Batching and much more, you might want to give it a try.
+
+### Dictionary
+
+Ternoa Dictionary records all the native substrate on-chain data of the Ternoa blockchain: blocks, extrinsics, and events. It is a glossary of data that pre-indexes chain events, drastically improving the overall indexing performance.
+
+The Dictionary acts as the middleman between the Blockchain and the Indexer. This allows the Indexer to query a block’s metadata from the dictionary, allowing one to query blocks for specific events and only return the required blocks. For example : If one wants to fetch NFT creation events only, the Indexer will ask the dictionary for the corresponding blocks, returning only the blocks where NFT creation took place (for example : 5, 9, 32, etc.) instead of all of the blocks (lets say 1 - 100).
+
+### Error Reporting
+
+If you encounter any errors along the way, technical or otherwise. Let us know and we'll deal with it swiftly.
+It'll help us further improve the overall experience for our users.
+
+- Open a discussion of type `General` in the [discussions section](https://github.com/capsule-corp-ternoa/ternoa-js/discussions) if you encounter any unexpected behaviour.
```

Comment:
Links are wrong, they are related to the TernoaJS SDK repo. Use these ones:

- Open a discussion of type `General` in the [discussions section](https://github.com/capsule-corp-ternoa/ternoa-subql-dictionary/discussions) if you encounter any unexpected behaviour.
- Open a Bug report submitting an [issue](https://github.com/capsule-corp-ternoa/ternoa-subql-dictionary/issues) if the bug persists.

Reply:
😐